openapi/reputation/reputation.yaml

```diff
@@ -139,6 +139,11 @@ paths:
         in: path
         required: true
         description: The ID of a business location review
+      - schema:
+          type: string
+        in: query
```

Comment:
Should this be in the path like the review ID? I guess it's not an action on the business, but on the review, so maybe not.

Also do we need to do something to mark it as required?

Reply:
I can make it required. Not sure what the stance on mixed path and query params is though.